### DISCUSSION TITLE
Preference - switch to output / result tab

### DISCUSSION THREAD

**Discussion by maciey** [dbeaver/dbeaver#15136] [Q&A] [answered]
I have lot of procedures where Im actually more interested **output** then **results** while Im executing q-ty

Would be great to have option do not auto switch to results, but output if user decides 
Or (even better) option to undock / dock somewhere else  output window so I can see both..

![obraz](https://user-images.githubusercontent.com/5410448/146556924-29e80ef4-54b3-4cec-990c-f814433263f2.png)

**Comment by Matvey16** [comment, marked as answer]
@maciey You can configure the output window to be separate from resilts window and to automatically open when new message is sent. Do you think this functionality will be enough?

![kso76uAOVl](https://user-images.githubusercontent.com/82543000/147648549-285a1cdf-f54e-4a85-9412-29ed885e310f.gif)

**Comment by maciey** [comment]
This is excellent. I didnt know about this option.
Closing this item then
